installer/install.sh

```diff
@@ -67,6 +67,22 @@ function check_passed_msg() {
 }
 
 function check_docker() {
+  check_exists_msg "Docker" 
+  command -v docker &>/dev/null
+  docker_exists=${?}; if [[ ${docker_exists} -ne 0 ]]; then
+    does_not_exist_msg "Docker" "https://www.docker.com/get-docker"
+    exit 1
+  fi
+  check_passed_msg "Docker"
+
+  echo -e "Checking Docker is running"
+  command ps -A | grep docker | grep -v grep &>/dev/null
```

Comment:
For me this command will have a return code of 0 when Docker is not running as I have vscode plugins for docker that show up in grep. I wonder if just attempting a `docker ps` would be enough here as it's return code when it cannot connect to the docker daemon is 1? What do you think?